src/sonic-yang-models/tests/yang_model_tests/tests/acl.json

```diff
@@ -0,0 +1,60 @@
+{
+    "ACL_RULE_UNDEFINED_PACKET_ACTION": {
+	"desc": "Configure undefined packet_action in ACL_RULE table.",
+	"eStr": ["Invalid value", "PACKET_ACTION"]
+    },
+    "ACL_TABLE_EMPTY_PORTS": {
+	"desc": "Configure ACL_TABLE with empty ports.",
+	"eStr": []
```

Comment:
Instead, we can say None, and change the test code to Handle None as []

Reply:
Removed eStr from the json file in this case and treated absence as []